package.json

```diff
@@ -1,27 +1,57 @@
 {
-  "name": "xnft-quickstart",
-  "version": "2.0.0",
-  "description": "",
-  "main": "index.js",
-  "author": "",
-  "license": "ISC",
+  "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "build": "npx xnft native build",
     "start": "npx xnft native start",
-    "dev": "npx xnft native dev"
-  },
-  "devDependencies": {
-    "@coral-xyz/xnft-cli": "latest"
+    "build": "expo export:web",
```

Comment:
i think this needs to be `expo export --platform web`

Reply:
Yea so the new bundler uses metro which we can use once we have multi-file support. 
Webpack allows for easer customization to bundle everything into 1 file for now.